lib/node_modules/@stdlib/array/zeros-like/docs/types/index.d.ts

```diff
@@ -632,10 +46,10 @@ declare function zerosLike( x: Uint8ClampedArray, dtype?: DataType ): Uint8Clamp
 * var y = zerosLike( x );
 * // returns [ 0, 0 ]
 */
-declare function zerosLike( x: Array<any>, dtype?: DataType ): Array<number>;
+declare function zerosLike<T extends AnyArray>( x: T ): T;
```

Comment:
As above, for a "generic" array `x`, we should always return `Array<number>`, correct? Here, if `x` is `Array<string>`, then, based on my reading, the returned array type is also `Array<string>`, which is not correct.

Reply:
Addressed.